src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.html

```diff
@@ -0,0 +1,22 @@
+<h1 mat-dialog-title class="title-center">Select Assessment Year</h1>
+<br>
+<div mat-dialog-content>
+  <form [formGroup]="selectForm
+  " (ngSubmit)="navigateToSelectedClientMaster()">
+    <div class="form-group">
+      <mat-form-field>
+        <mat-label>Assessment Year</mat-label>
+        <mat-select name="assessmentYear" formControlName="selected">
+          <mat-option *ngFor="let year of assessmentYears" [value]="year.value">
```

Comment:
what if year is empty ? will that never occur ?

Reply:
Year for now is hardcoded. Later we will see how can we fetch it